opensrp-giz-malawi/src/main/java/org/smartregister/giz/repository/GizMalawiRepository.java

```diff
@@ -190,6 +189,11 @@ public void onUpgrade(SQLiteDatabase db, int oldVersion, int newVersion) {
                     break;
                 case 13:
                     upgradeToVersion13CreateReasonForDefaultingTable(db);
+                    break;
+                case 14:
```

Comment:
I think you should add this to 13 because of https://github.com/OpenSRP/opensrp-client-giz-malawi/pull/367

Reply:
So, I merge all the Migrations to 13 or Move 13 to 14 and Vice versa